cmd/export_ledger_entry_changes.go

```diff
@@ -99,9 +102,18 @@ be exported.`,
 							if err != nil {
 								entry, _, _, _ := utils.ExtractEntryFromChange(change)
 								cmdLogger.LogError(fmt.Errorf("error transforming account entry last updated at %d: %s", entry.LastModifiedLedgerSeq, err))
+							} else {
+								transformedOutputs["accounts"] = append(transformedOutputs["accounts"], acc)
+							}
```

Comment:
❓  Should this be wrapped in an `else` statement? The rest of the `LedgerEntryType` append the `LedgerType` regardless if an error is returned.

Reply:
I think in other instances where there is an error during `Transform*`, the error check hits `continue`, and the next `ledgerEntry` is considered. 

In this case, both `signers` and `account` depend on the same `ledgerEntry`. I guess it depends on whether we still want to try to extract signers if `TransformAccount` returns an error. I think it's reasonable to just bail as well.